azerothcore/azerothcore-wotlk
# Fix(DB/Quest): Remove Nipsy Handin location

The quest location is showing in an incorrect place. As the handin is in deeprun tram, the quest location has been removed.



## Changes Proposed:
-  Removed quest location from the quest_poi and quest_poi_points tables, as it is in the deeprun tram.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/9086

## SOURCE:


## Tests Performed:

-  Tested in game by checking the map on Windows. 


## How to Test the Changes:


1. Get an alliance character at level 10
2. .q add 6662
3. Select quest on log and click on "Show Map" or simply Open map to see where POI for quest is located.
4. Where previously it would show the quest as being in silverpine forest, it should no longer show a quest marker at all. 
5. EDIT:: It should now show the quest as being in the deeprun tram entrance in either stormwind or ironforge, depending on whichever is closest.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
